scenario:
  name: 'Scenario Camille'

  overview:  " 0. Starting from -35km below the target do ARM1.\n
              1. Do ARM2 and CAM2.\n
              2. Do ARM3 to go 4km below the target.\n
              3. Drift to S0.\n
              4. Do ARM1 to move on target's orbit.\n"

  keep_out_zone: 0.2

  approach_ellipsoid: [0.5, 1.5, 0.5]

  CheckPoints:
    S0:
      id: 0
      position:
        kep:
          a: target.a - 47.0
          e: target.a * target.e / self.abs_state.a
          i: target.i
          w: target.w
          O: target.O

    S1:
      id: 1
      position:
        kep:
          a: target.a - 16.0
          e: target.a * target.e / self.abs_state.a

    S2:
      id: 2
      position:
        lvlh:
          R: [-4.0, -10.0, 0.0]
      error_ellipsoid: [0.4, 1.5, 0.4]
      manoeuvre: drift

    S3:
      id: 3
      position:
        lvlh:
          R: [-4.0, 8.0, 0.0]
      error_ellipsoid: [0.4, 1.5, 0.4]
      manoeuvre: drift

    S4:
      id: 4
      position:
        lvlh:
          R: [0.0, 18.0, 0.0]
      error_ellipsoid: [0.2, 1.0, 0.2]
      t_min: 3000
      t_max: 3100

    S5:
      id: 5
      position:
        lvlh:
          R: [0.0, -2.0, 0.0]
      error_ellipsoid: [0.1, 0.3, 0.1]
      t_min: 29000
      t_max: 30000

    S6:
      id: 6
      position:
        lvlh:
          R: [0.0, -1.0, 0.0]
      error_ellipsoid: [0.05, 0.15, 0.05]
      manoeuvre: radial